config/develop/namespaced/glue-job-S3ToJsonS3.yaml

```diff
@@ -1,4 +1,5 @@
 #{% set ns = stack_group_config.namespace %}
+#{% set glue_script_tag = 'ETL-91' %}
```

Comment:
The glue_script_tag above we will change to an acceptable tag before it reaches prod. If the scripts were in a separate repository they would go through review, be merged to main, then tagged. Then in the infra repository, we would refer to the tagged version of the scripts. In this repo since we're trying to do everything in one place, the workflow will look something like:
1. do work on a branch for dev
2. merge to main
3. tag
4. fix up prod config including referencing correct tag for scripts
5. merge to main again

Eventually, we'll split all of this into separate repositories for simplicity and cleanliness.